sympy/series/series.py

```diff
@@ -10,3 +13,133 @@ def series(expr, x=None, x0=0, n=6, dir="+"):
     """
     expr = sympify(expr)
     return expr.series(x, x0, n, dir)
+
+
+def lagrange_inversion_theorem(eq, no_of_terms = 3):
```

Comment:
No space around `=` when used for keyword arguments. 

Also, a better name would just be `n`, which is what `series` uses.

Reply:
Done now, please take a look